app/lib/Plugins/SearchEngine/Elastic8/Mapping.php

```diff
@@ -273,50 +273,50 @@ public function getConfigForElement($ps_table, $pn_element_id, $pa_element_info,
         $va_element_config[$ps_table . '/' . $vs_element_code . '_end']['type'] = 'date';
         $va_element_config[$ps_table . '/' . $vs_element_code . '_end']['ignore_malformed'] = true;
         break;
-      case 4:  // geocode
+      case __CA_ATTRIBUTE_VALUE_GEOCODE__:
         //@see https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-geo-shape-type.html
         $va_element_config[$ps_table . '/' . $vs_element_code] = array(
           'type' => 'geo_shape',
         );
         // index text content as is -- sometimes useful for full text place search
         $va_element_config[$ps_table . '/' . $vs_element_code . '_text'] = array('type' => 'text');
         break;
-      case 6: // currency
+      case __CA_ATTRIBUTE_VALUE_CURRENCY__:
         // we want to do range searches on currency too, so we gotta store the currency identified (USD) separately
         $va_element_config[$ps_table . '/' . $vs_element_code]['type'] = 'double';
         $va_element_config[$ps_table . '/' . $vs_element_code . '_currency'] = array('type' => 'text');
         break;
-      case 8: // length
-      case 9: // weight
+      case __CA_ATTRIBUTE_VALUE_LENGTH__:
+      case __CA_ATTRIBUTE_VALUE_WEIGHT__:
         // we don't index units here -- we always index in meters / kg, so it's just a float
         $va_element_config[$ps_table . '/' . $vs_element_code]['type'] = 'double';
         break;
-      case 10:  // timecode
-      case 12:  // numeric/float
+      case __CA_ATTRIBUTE_VALUE_TIMECODE__:
+      case __CA_ATTRIBUTE_VALUE_NUMERIC__:
         $va_element_config[$ps_table . '/' . $vs_element_code]['type'] = 'double';
         break;
-      case 11:  // integer
+      case __CA_ATTRIBUTE_VALUE_INTEGER__:
         $va_element_config[$ps_table . '/' . $vs_element_code]['type'] = 'long';
         break;
-      case 1: // text
-      case 3:  // list
-      case 5:  // url
-      case 13: // LCSH
-      case 14: // geonames
-      case 15: // file
-      case 16: // media
+      case __CA_ATTRIBUTE_VALUE_TEXT__:
+      case __CA_ATTRIBUTE_VALUE_LIST__:
+      case __CA_ATTRIBUTE_VALUE_URL__:
+      case __CA_ATTRIBUTE_VALUE_LCSH__:
+      case __CA_ATTRIBUTE_VALUE_GEONAMES__:
+      case __CA_ATTRIBUTE_VALUE_FILE__:
+      case __CA_ATTRIBUTE_VALUE_MEDIA__:
       case 19: // taxonomy
```

Comment:
@kehh There does not seem to be an implementation for Taxonomy? Am investigating

Reply:
Looks like they've replaced those with InformationService which is good. Let's kill this case

Reply:
(especially as there's a default which does the same thing)